public/src/js/script.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * 
+ * Displays information about the method been called.
+ * @param {String} id - id of method to display
+ * @returns {void}
+ */
+const showMethodDetail = (id) => {
+  $('.details pre div').map((index, domElement) => {
+    domElement.style.display = 'none';
```

Comment:
Since you're using jquery in some parts, why not just use it completely, especially for dom manipulations task, you can use `.hide()` in this case